<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSSaveFileClass</class>
 <widget class="QMainWindow" name="DSSaveFileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DSSaveFile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="profile_combo_box">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="profile_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renameButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>rename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>360</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="game_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Game:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="game_combo_box">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dark Souls III</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sekiro: Shadows Die Twice</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="save_file.qrc"/>
 </resources>
 <connections/>
</ui>
